src/main/java/it/pagopa/pm/gateway/utils/VPosUtils.java

```diff
@@ -0,0 +1,62 @@
+package it.pagopa.pm.gateway.utils;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Component
+public class VPosUtils {
+    public static final int ID_PSP_POSITION = 0;
+    public static final int ABI_POSITION = 1;
+    public static final int SHOP_ID_FIRST_PAY_POSITION = 2;
+    public static final int TERMINAL_ID_FIRST_PAY_POSITION = 3;
+    public static final int MAC_FIRST_PAY_POSITION = 4;
+    public static final int SHOP_ID_NEXT_PAY_POSITION = 5;
+    public static final int TERMINAL_ID_NEXT_PAY_POSITION = 6;
+    public static final int MAC_NEXT_PAY_POSITION = 7;
+    public static final String RESULT_CODE_AUTHORIZED = "00";
+    public static final String RESULT_CODE_METHOD = "25";
+    public static final String RESULT_CODE_CHALLENGE = "26";
+    private static final String ASTERISK_SPLIT_CHAR = "\\*";
+    private static final String PIPE_SPLIT_CHAR = "\\|";
+    private final Map<String, List<String>> vposShopMap = new HashMap<>();
+
+    @Value("${vpos.vposShops}")
+    private String vposShops;
+
+    /*
+     * A VPos shop is identified by the following parameters:
+     * idPsp, ABI, shopId, terminalId, MAC
+     * where shopId, terminalId and MAC are different for first (F) or subsequent (S) payments.
+     * Each property in the config environment property is separated by a | and each shop is separated by a *
+     * following this schema:
+     * idPsp|ABI|shopId(F)|terminalId(F)|MAC(F)|shopId(S)|terminalId(S)|mac(S)*idPsp|... ...terminalId(S)|mac(S)
+     */
```

Comment:
It's an environment variable, so it must be saved in the azure properties and this seem to us the best method to save this variable.
I don't know if it can be saved as json, let me know